templates/home_page.hbs

```diff
@@ -17,24 +17,38 @@
   <div class="container">
     <section class="section knowledge-base">
       <section class="categories blocks">
+      <h1 class="geo-medium" style="font-size: 40px">Consumer Products</h1>
```

Comment:
Maybe `Consumer Products` should be in the list of translations. Could be added as Dynamic Content.

Reply:
Got it, I'll put it as {{dc 'consumer_products'}} for now.